_includes/pbc/all/install-features/202410.0/enable-dynamic-multistore.md

```diff
@@ -0,0 +1,56 @@
+This document describes how to enable [Dynamic Multistore](/docs/pbc/all/dynamic-multistore/{{page.version}}/base-shop/dynamic-multistore-feature-overview.html) on the latest codebase.
+
+## Enable Dynamic Multistore
+
+{% info_block warningBox "Project version" %}
+If your project version is below 202307.0, you need to update to the latest codebase first, see [Install Dynamic Multistore](/docs/pbc/all/dynamic-multistore/202410.0/base-shop/install-dynamic-multistore.html) page.
```

Comment:
What do I need to do with the linked document?

Reply:
Rephrased, please take a look.
I would prefer not to merge those documents, this is a different things.